internal/provider/data_source_project.go

```diff
@@ -58,6 +58,12 @@ func dataSourceTFEProject() *schema.Resource {
 				Computed: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+
+			"effective_tags": {
+				Type:     schema.TypeMap,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
```

Comment:
For the reviewer: Thinking about this a little more, workspace/project data sources should simply have `tags` be the list of effective tags and leave direct tag bindings for each resource. Therefore there would be no `effective_tags` attribute on these. Wdyt?